### ISSUE TITLE
Active directory integration

### ISSUE THREAD

**Issue by firozhaneef** [snipe/snipe-it#1121]
Hi,

I have installed v2.0 and started using it, later on i came to understand that it has AD integration feature. How do i enable this feature, Do i have to upgrade after i edit and fill the correct information of our AD in ldap.php file?

Firoz

**Comment by snipe** [comment]
http://docs.snipeitapp.com/installation/configuration.html#optional-ldap-configuration

I'm not sure what you mean by upgrade after you edit the LDAP configuration file. If you're running 2.0, there's nothing to upgrade to. That's the latest release.

**Comment by firozhaneef** [comment]
what i mean, if i edit the ldap.php file and provide the ldap settings, and simply access the users page, will it be enough to download the users from AD or do i have to make any other additional set-up to activate this feature?

**Comment by snipe** [comment]
Once there is an ldap.php in your config directory, the LDAP import button will appear on the users page.

**Comment by firozhaneef** [comment]
Thanks for such a nice work, i love this software.

**Comment by snipe** [comment]
![mogwai-dance](https://cloud.githubusercontent.com/assets/197404/9505706/cc8a9f6a-4bf8-11e5-8471-64bf4150b934.gif)

**Comment by ExactCare** [comment]
I have configured LDAP and it connects ok, However, when I try to "Synchronize" it gives me an error on import. I have included a screen shot of the error in the notes area.

![image](https://cloud.githubusercontent.com/assets/14076881/9634005/9a99da74-515d-11e5-8022-9585ea515132.png)

I am using Active Directiory and the fields for first name and username are set as: 

'result.username' => "sAMAccountName",
 'result.last.name'  => "sn",
 'result.first.name' => "givenName",


Also are there additional fields for people that we can populate with info from ldap?
